src/components/DataViewer.vue

```diff
@@ -214,7 +214,139 @@ export default class DataViewer extends Vue {
 </script>
 <style lang="scss" scoped>
 @import '../styles/_variables';
-@import '../styles/DataViewer';
+
+.data-viewer {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  @extend %main-font-style;
+  /deep/::after,
+  /deep/::before {
+    box-sizing: border-box;
+  }
+
+  /deep/button {
```

Comment:
C'est pas forcément obligatoire mais je te conseille de mettre un espace après le selecteur /deep/ quand c'est pas un pseudo-element
```suggestion
  /deep/ button {
```